inference-engine/src/cldnn_engine/cldnn_engine.cpp

```diff
@@ -279,6 +280,7 @@ InferenceEngine::CNNNetwork clDNNEngine::CloneAndTransformNetwork(const Inferenc
             pass_config->disable<ngraph::pass::SoftPlusDecomposition>();
             pass_config->disable<ngraph::pass::LogSoftmaxDecomposition>();
             pass_config->disable<ngraph::pass::ConvertBroadcast3>();
+            pass_config->disable<ngraph::pass::MVN6Decomposition>();
```

Comment:
If MVN op has unsupported axes combination, then we'll have an exception which is not an expected behavior.
So instead of disabling this transformation you should set callback which keeps MVN-6 for supported cases and decompose it for unsupported ones.